components/job-orchestration/job_orchestration/executor/query/utils.py

```
@@ -134,3 +155,32 @@ def update_query_task_metadata(
+    archive's ``uncompressed_size`` from the metadata database.
+    """
+    try:
+        with (
+            closing(sql_adapter.create_connection(True)) as db_conn,
+            closing(db_conn.cursor(dictionary=True)) as db_cursor,
+        ):
+            table_name = get_archives_table_name(
+                clp_metadata_db_conn_params["table_prefix"], dataset
+            )
+            db_cursor.execute(
+                f"SELECT uncompressed_size FROM {table_name} WHERE id = %s",
```

Ruff (0.15.13)

[error] Possible SQL injection vector through string-based query construction

(S608)